apps/platform/src/hooks/usePermissions.js

```diff
@@ -0,0 +1,7 @@
+import config from '../config';
+
+function usePermissions() {
+  return config.profile.isPartnerPreview;
+}
```

Comment:
Please expose a variable from the hook to make it more clear

Change:
```
isPartnerPreview = usePermissions()
```
to:

```
const { isPartnerPreview } = usePermissions()
```

Reply:
done